<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqScatterPlotDisplayPanel</class>
 <widget class="QWidget" name="pqScatterPlotDisplayPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>1368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="ViewGroup">
     <property name="title">
      <string>View</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ViewData">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Toggle visibility of this dataset's geometry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Visible</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ViewZoomToData">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Change the camera location to best fit the dataset in the view window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Zoom To Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="Selectable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;Toggle the selectablity of the object. Only selectable objects can be selected using the selection mechanisms&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Selectable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="InputsGroupBox">
     <property name="title">
      <string>Inputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="XCoordsLabel">
        <property name="text">
         <string>X Coords</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="XCoordsComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="YCoordsLabel">
        <property name="text">
         <string>Y Coords</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="YCoordsComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ZCoordsCheckBox">
        <property name="text">
         <string>Z Coords</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="ZCoordsComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="ColorCheckBox">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="ColorComboBox"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="GlyphGroupBox">
        <property name="title">
         <string>Glyph</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QCheckBox" name="GlyphScalingCheckBox">
           <property name="text">
            <string>Scaling</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="GlyphScalingComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="GlyphMultiSourceCheckBox">
           <property name="text">
            <string>Multi Source</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="GlyphMultiSourceComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="GlyphOrientationCheckBox">
           <property name="text">
            <string>Orientation</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="GlyphOrientationComboBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="GlyphScaleFactorLabel">
           <property name="text">
            <string>Scale Factor</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="GlyphScaleFactorSpinBox">
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ColorGroup">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="ColorInterpolateScalars">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If on, scalars will be interpolated within polygons and color mapping will happen on a per-pixel basis.  If off, color mapping occurs at polygon points and colors are interpolated, which is generally less accurate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Interpolate Scalars</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="EditColorMapButton">
        <property name="text">
         <string>Edit Color Map...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="RescaleButton">
        <property name="text">
         <string>Rescale to Data Range</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ColorMapScalars">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Map scalars through lookup table before applying color.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Map Scalars</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="AnnotationGroup">
     <property name="title">
      <string>Annotation</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="EditCubeAxes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;Edit properties of the cube axes &lt;/html&gt;</string>
        </property>
        <property name="statusTip">
         <string>&lt;html&gt;Edit properties of the cube axes &lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;Edit properties of the cube axes &lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ShowCubeAxes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;Toggle visibility of the cube axes&lt;/html&gt;</string>
        </property>
        <property name="statusTip">
         <string>&lt;html&gt;Toggle visibility of the cube axes&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;Toggle visibility of the cube axes&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Show cube axes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="StyleGroup">
     <property name="title">
      <string>Style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="3">
       <widget class="QTreeWidget" name="compositeTree">
        <column>
         <property name="text">
          <string>Block selection:</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Line width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="StyleLineWidth">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If your dataset containes lines/edges, this scale adjusts the width of the rendered lines.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>253</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Point size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="StylePointSize">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If your dataset contains points/vertices, this scale adjusts the diameter of the rendered points.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="Opacity">
        <property name="toolTip">
         <string>&lt;html&gt;Set the opacity of the dataset's geometry.  Artifacts
         may appear in translucent geometry because primatives are not
         sorted.&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ViewData</tabstop>
  <tabstop>ViewZoomToData</tabstop>
  <tabstop>Selectable</tabstop>
  <tabstop>ColorInterpolateScalars</tabstop>
  <tabstop>ShowCubeAxes</tabstop>
  <tabstop>EditCubeAxes</tabstop>
  <tabstop>compositeTree</tabstop>
  <tabstop>StylePointSize</tabstop>
  <tabstop>Opacity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ShowCubeAxes</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditCubeAxes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorMapScalars</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditColorMapButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorMapScalars</sender>
   <signal>toggled(bool)</signal>
   <receiver>RescaleButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
